### ISSUE TITLE
[BUG] base16 colorscheme not applied

### ISSUE THREAD

**Issue by r3n3gad3p3arl** [nix-community/nixvim#1264]
| Field        | Description |
|--------------|-------------|
| Plugin       | `base16`  | 
| Nixpkgs      | `unstable` | 
| Home Manager | `unstable` | 

- [x] I have read the [FAQ](https://nix-community.github.io/nixvim/user-guide/faq.html) and my bug is not listed there.

## Description
When the base16 colorscheme is enabled as seen in the MRE, it does not apply itself as the Neovim colorscheme. Strangely, it _does_ apply to lualine. I've tried multiple base16 colorschemes, all of them fail to apply. Possibly related to #1189 or #1228 (though the onedark plugin works fine for me).

## Minimal, Reproducible Example (MRE)

```nix
programs.nixvim = {
  enable = true;
  colorschemes.base16 = {
    enable = true;
    colorscheme = "tomorrow-night";
  };
}
```

**Comment by GaetanLepage** [comment]
This is weird indeed.
Looking at the generated `init.lua`, `colorscheme base16-tomorrow-night` is correctly appended to `extraConfigVim`...

**Comment by r3n3gad3p3arl** [comment]
I figured it out! Just opened a pull request to fix the issue.

**Comment by GaetanLepage** [comment]
Fixed by #1265
